project/src/main/webapp/shared.js

```diff
@@ -213,3 +213,14 @@ function buildSkillsAsLabels(querySelector, opportunities) {
     buildAsLabels(querySelector, opportunity.requiredSkills, 'skills');
   }
 }
+
+/**
+ * Adds currently-attributed profile image to logged-in user
+ * If the user has no profile image, add the default one
+ */
+async function populateExistingProfileImage() {
+  const response = await fetch('/blob-handler');
```

Comment:
Im wondering if you need to pass a user id here to get the image for this specific user.

Reply:
I just did it for the current logged-in user, because our app doesn't currently have the functionality to view other profiles

Reply:
I'm thinking that if we have a feature next week that involves showing profiles, this function would be updated there instead

Reply:
Understood, makes sense.